samples/event_manager_profiler/src/events/five_sec_event.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <stdio.h>
+#include "five_sec_event.h"
+
+
+static void profile_five_sec_event(struct log_event_buf *buf,
+				  const struct event_header *eh)
+{
+}
+
+EVENT_INFO_DEFINE(five_sec_event,
+		  ENCODE(),
+		  ENCODE(),
+		  profile_five_sec_event);
+
+EVENT_TYPE_DEFINE(five_sec_event,
+		  true,
```

Comment:
Maybe we should actually log the events too? Thanks to that user will be able to see events both in logs and in profiler.

Reply:
Alternatively set this field to false if there is no function to log event (that is more intuitive). Also for other events.

Reply:
Sorry do not understand.
Those events do not have any data to be logged. And if we turn it off we would not see inn terminal at all that the event had occurred.

Reply:
Ok, I overlooked the fact that we still log event if the logging function is set to NULL. Sorry. Leave it as it is
Some time ago it worked in an other way.